src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -1623,6 +1623,7 @@ void NetworkOPsImp::pubValidation (STValidation::ref val)
             val->getSignerPublic());
         jvObj [jss::ledger_hash]           = to_string (val->getLedgerHash ());
         jvObj [jss::signature]             = strHex (val->getSignature ());
+        jvObj [jss::validation_type]       = val->isFull() ? "full" : "partial";
 
```

Comment:
Would the message always include both `full` and `partial` with one marked `true` and the other `false`?